Content.Shared/Anomaly/Effects/Components/EntitySpawnAnomalyComponent.cs

```diff
@@ -14,13 +14,19 @@ public sealed partial class EntitySpawnAnomalyComponent : Component
     [DataField("spawns", required: true, customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>)), ViewVariables(VVAccess.ReadWrite)]
     public List<string> Spawns = new();
 
+    [DataField("superCriticalSpawns", required: true, customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>)), ViewVariables(VVAccess.ReadWrite)]
+    public List<string> SuperCriticalSpawns = new();
```

Comment:
you can just replace this with a list of `EntProtoId`s

Reply:
No idea how to use EntProtoId and have it tie in with the way the entities are spawned in the system.

Reply:
this has been done.